t/18-qemu.t

```diff
@@ -426,9 +426,9 @@ subtest 'qemu is not called on an empty file when ISO_1 is an empty string' => s
     my $mock_proc = Test::MockModule->new('OpenQA::Qemu::Proc');
     my $call_count = 0;
     $mock_proc->redefine(get_img_size => sub {
-            my ($iso) = @_;
-            $call_count++;
-            die 'get_img_size called on an empty string' unless $iso;
+            my ($iso) = @_;    # uncoverable statement
```

Comment:
Hm, if a mocked function is never called, maybe it doesn't need to be mocked at all.
I just ran the test without this `redefine`, and it passed.

Reply:
Funnily, this was added by me, so it seems I overlooked something.

Reply:
I think this function is called -> if tests fail :D